src/librustc_yk_sections/emit_tir.rs

```diff
@@ -11,15 +11,14 @@
 //!
 //! Serialisation itself is performed by an external library: ykpack.
 
+// FIXME, just temporarily.
```

Comment:
Why?

Reply:
> I've left some unused code behind that I know I will need to adapt later, hence the #![allow(...)].

Reply:
Yes, but what code? And can we briefly explain why it's temporary? I always worry these things become more permanent than expected!

Reply:
Every field in the conversion context is currently unused, but we will certainly use them later.

I went the `allow` route instead of underscoring all the variable names, as I thought it would be easier to review.

Reply:
To expand on that:

For example the next variable counter is unused since there are no conversions to make new variables.

Reply:
Is this going to be fixed very soon (e.g .the next PR)?

Reply:
I hope so, yes!